src/services/transaction-service.js

```diff
@@ -0,0 +1,20 @@
+import { StorageService } from "./storage";
+
+export class TransactionListService {
+  static get() {
+    return StorageService.get("transactionList");
+  }
+
+  static set(value) {
+    return StorageService.set("transactionList", value);
+  }
+
+  static add(data) {
+    return this.get()
+      .then(list => {
+        const newList = list ? [data].concat(list) : [data];
+        this.set(newList);
+      })
+      .catch(e => console.error(`get transactions: ${e.message}`));
```

Comment:
please remove all `catch` statements in your code where you do not do anything but `console.log/error`

Reply:
there is no need for that